website/docs/user-guide/concepts/external-services.md

```diff
@@ -0,0 +1,102 @@
+# External Services
+
+This document describes how to provide external services to ORT runs, like for instance source code or artifact repositories.
```

Comment:
Suggestion: "... how to configure credentials for external services, like source code or artifact repositories."